datareservoirio/client.py

```diff
@@ -250,11 +250,18 @@ def wrapper(self, series_id, start=None, end=None, **kwargs):
             result = func(self, series_id, start=start, end=end, **kwargs)
             end_time = time.perf_counter()
             elapsed_time = end_time - start_time
+            format = "%Y-%m-%DT%H:%M:%S.%f%z"
+            start_date_as_str = pd.to_datetime(
+                start, dayfirst=True, unit="ns", utc=True
+            ).strftime(format)
+            end_date_as_str = pd.to_datetime(
+                end, dayfirst=True, unit="ns", utc=True
+            ).strftime(format)
```

Comment:
Looks to me like the format does not work as expected. What about just using the `.isoformat()` method? 
![image](https://user-images.githubusercontent.com/68221790/227889522-c1f866ca-80f6-479c-b94f-e1dd3d36c572.png)

Reply:
Updated the code to use isoformat()